includes/Experiments.php

```diff
@@ -369,6 +369,17 @@ public function get_experiments(): array {
 				'group'       => 'editor',
 				'default'     => true,
 			],
+			/**
+			 * Author: @sblinde
+			 * Issue: #5326
+			 * Creation date: 2022-06-29
+			 */
+			[
+				'name'        => 'showSemanticHeadings',
+				'label'       => __( 'Semantic headings', 'web-stories' ),
+				'description' => __( 'Display semantic heading tags', 'web-stories' ),
```

Comment:
Slightly misleading because we already do use semantic heading tags; this is just about exposing a UI for them.

Maybe something like this:

```suggestion
				'description' => __( 'Display UI for changing semantic heading tags', 'web-stories' ),
```

Not a big deal though since this will be a rather short-lived feature flag

Reply:
Awesome! Did just that, thank you! https://github.com/GoogleForCreators/web-stories-wp/pull/11888/commits/0406524707b2ba47c41ac958eaccac4567c1d98d